### PR TITLE
Added change logs for new minor releases

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: src/Core/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
 # Change Log
 
-## 0.3.4
+## 0.4.0
+
+### Added
+
+- `AsyncAws\Core\Request::getQuery()`
```
Comment (jderusse) [inline src/Core/CHANGELOG.md]: The class is internal, do we have to mention theme in changlog?

File: src/Core/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+- Added constructor argument to  `AsyncAws\Core\Request::__construct()` to support query parameters.
+- Renamed `AsyncAws\Core\Request::getUrl()` to `AsyncAws\Core\Request::getEndpoint()`
+- Class `AsyncAws\Core\Stream\StreamFactory` is not internal anymore.
+- Removed `requestBody()`, `requestHeaders()`, `requestQuery()` and `requestUri()` input classes. They are replaced with `request()`.
+
+### Removed
+
+- Public `AbstractApi::request()` was removed.
+- Protected function `AbstractApi::getEndpoint()` was made private.
+- Private function `AbstractApi::fillEndpoint()` was removed.
```
Comment (jderusse) [inline src/Core/CHANGELOG.md]: same comment for private methods

File: src/Service/CloudFormation/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
 
+## 0.3.0
+
+### Added
+
+- Enums; `Capability`, `ResourceStatus`, `StackDriftStatus`, `StackStatus`
+
+### Changed
+
+- Removed `requestBody()`, `requestHeaders()`, `requestQuery()` and `requestUri()` input classes. They are replaced with `request()`.
```
Comment (jderusse) [inline src/Service/CloudFormation/CHANGELOG.md]: Internal methods

Comment (Nyholm) [reply src/Service/CloudFormation/CHANGELOG.md]: There were not internal at last release.

Comment (jderusse) [reply src/Service/CloudFormation/CHANGELOG.md]: :+1:

**Pull request merged**